File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ItemPhysics.java
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge by LiquidBounce.
 * https://github.com/UnlegitMC/FDPClient/
 */
package net.ccbluex.liquidbounce.features.module.modules.render;

import net.ccbluex.liquidbounce.features.module.*;
import net.ccbluex.liquidbounce.value.FloatValue;

@ModuleInfo(name = "ItemPhysics", category = ModuleCategory.RENDER)
public class ItemPhysics extends Module  {
    public final FloatValue itemWeight = new FloatValue("Weight", 0.5F, 0F, 1F);

    @Override
    public String getTag() {
        return itemWeight.get().toString();
    }
}

File: src/main/java/net/skiddermc/fdpclient/features/module/modules/render/ItemPhysics.kt
/*
 * FDPClient Hacked Client
 * A free open source mixin-based injection hacked client for Minecraft using Minecraft Forge by LiquidBounce.
 * https://github.com/SkidderMC/FDPClient/
 */
package net.skiddermc.fdpclient.features.module.modules.render

import net.skiddermc.fdpclient.features.module.*
import net.skiddermc.fdpclient.value.FloatValue

@ModuleInfo(name = "ItemPhysics", category = ModuleCategory.RENDER)
class ItemPhysics : Module() {
    val itemWeight = FloatValue("Weight", 0.5F, 0F, 1F)
    override val tag: String?
        get() = "${itemWeight.get()}"
}